src/openloco/ui/WindowManager.cpp

```diff
@@ -1440,7 +1440,8 @@ namespace openloco::ui::WindowManager
                 continue;
 
             scrollIndex++;
-            if (window->scroll_areas[scrollIndex].flags & 0b10001)
+            constexpr uint16_t scrollbarFlags = scrollview::scroll_flags::HSCROLLBAR_VISIBLE | scrollview::scroll_flags::VSCROLLBAR_VISIBLE;
```

Comment:
Move into the header with the name `ANY_SCROLLBAR_VISIBLE`? Its used a in a few locations.

Reply:
You mean as a mask? Sure, that sounds useful.